dev/devicelab/lib/framework/ios.dart

```diff
@@ -22,6 +22,8 @@ Future<bool> containsBitcode(String pathToBinary) async {
   // See: https://stackoverflow.com/questions/32755775/how-to-check-a-static-library-is-built-contain-bitcode
   final String loadCommands = await eval('otool', <String>[
     '-l',
+    '-arch',
+    'arm64',
```

Comment:
Now release add-to-app fat `Flutter.framework` contains debug x86 so it can be built on the simulator.  `otool -l` doesn't show the bitcode section for a fat framework if any slices don't contain bitcode.  Specify `arm64` only since that's the part that actually gets shipped and requires bitcode.

Reply:
What about armv7?

Reply:
This has mostly been useful as a sanity check to make sure a debug version wasn't incorrectly embedded, not that the builder is producing the right artifacts, so that was a shortcut.  I can check both if you prefer!